Ignore freud's generated Cython C++ files in git. (#594)
glotzerlab/freud: ChangeLog.md

and this project adheres to
[Semantic Versioning](http://semver.org/spec/v2.0.0.html).

## next

### Added
- Support for garnett 0.7.

### Changed

> - Cython is no longer an optional dependency. Cythonized `.cpp` files have been removed.

Author: ```suggestion
- Cython is now a required dependency (not optional). Cythonized `.cpp` files have been removed.
```
